pyproject.toml

```diff
@@ -13,7 +13,7 @@ numpy = "^1.18"
 scipy = "^1.8.0"
 tqdm = "^4.62.3"
 PyWavelets = "^1.2.0"
-
+numba = "^0.55.1"
 
 [build-system]
 requires = ["poetry-core"]
```

Comment:
Please change version number to 0.1.2 so that we can upload the updates version to pip
Also update version number in __init__.py